source/compiler-core/windows/slang-win-visual-studio-util.cpp

```diff
@@ -69,15 +69,15 @@ static SlangResult _readRegistryKey(const char* path, const char* keyName, Strin
 
 // Make easier to set up the array
 
-static DownstreamCompilerMatchVersion _makeVersion(int main)
+[[maybe_unused]] static DownstreamCompilerMatchVersion _makeVersion(int main)
```

Comment:
I think we have a macro define for maybe_unised.
Something like SLANG_MAYBE_UNUSED or something

Reply:
now that we're on c++17 we should avoid that macro